exercises/level8/README.md

```diff
@@ -1,15 +1,79 @@
+## LEVEL 8
+- Persitance
+- Volume
 
-Start :
-./workshopctl -n level8 apply -f level8/.backstage/level8.yml
+### Objectives
+
+- Undestanding data persitence
+- Learn about volumes in Kubernetes
+- Write and use a Persistent volume claim
+- Understand limits of volume
 
-check status
-./workshopctl -n level8 get all                              
+### Instructions
 
-Go to website, upload file
-http://upload.level8.localhost/
+- Start the level, `make level8.start` or 
+```bash
+# Start the level
+./workshopctl -n level8 apply -f level8/.backstage/level8.yml
+./workshopctl -n level8 apply -f level8/.backstage/local-path-provisionner.yml
+# Check status
+./workshopctl -n level8 get all
+```
 
-delete the pod to force a restart
+- Go to website (http://upload.level8.localhost/) and upload a file
+- Delete the pod to force a restart
+```bash
 ./workshopctl -n level8 delete pods --all
+```
+
+- Go to the website (http://upload.level8.localhost/) again, the file uploaded disappeared
+
+File or anything stored on the disk is not persisted in a restart, for than you we to put a persistant volume in place:
```

Comment:
`File or anything stored on the disk is not persisted in a restart, for than you need to put a persistent volume in place:`

Reply:
Fixed